presto-main/src/main/java/com/facebook/presto/sql/analyzer/Scope.java

```diff
@@ -31,7 +31,9 @@
 import static com.facebook.presto.sql.analyzer.SemanticExceptions.missingAttributeException;
```

Comment:
I thought that was the spec. If that is not the case and `WithQueries` are always visible along with fields, then the `Scope` code can be simplified a bit. Thanks!

FYI: @kokosing